pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -51,19 +51,24 @@ display(tile)
 
 ## URI Formats
 
-/[ TODO Populate this with valid URI's schemes we can use with an example? including how to use gdal /]
-
-file://  (is this only valid for spark local? or if file is )
-https://
-http:// ?
-s3a://
-s3n:// ?
-hdfs:// ?
-ftp:// ?
-azure ?
-google cloud ?
-
-gdal vsi? https://gdal.org/user/virtual_file_systems.html 
+RasterFrames relies on three different IO drivers, selected based on a combination of scheme, file extentions, and library availability. GDAL is used first if a compatible version of GDAL (>= 2.4) is installed. It is the only driver that can read non-GeoTIFF files. If GDAL is not available, either the _Java I/O` or _Hadoop_ driver will be selected, depending on scheme.  
```

Comment:
Possibly formatting issue here with Java I/O

Reply:
What do you mean?